content/topics/sharing-code-workflows.md

```diff
@@ -0,0 +1,84 @@
+Software encompasses code in a packaged, documented and operational form, designed to conduct a range of scientific tasks, thereby becoming an essential tool in the research project.
+
+**Workflows** 
+
+Scientific workflows or pipelines describe the structured sequences that organise and automate research tasks. They detail the flow of data between tasks, often managed by Workflow Management Systems (WMS), and encapsulate a higher level of complexity by integrating multiple software components, scripts, and tools. 
+
+## Why share code & workflows?
+<div class="row">
+  <div class="col-7">
+    In life science research, code and workflows are often as important as the data itself. From data processing scripts to full analysis pipelines — sharing your code is about <b>openness</b>, <b>improving</b> your science, gaining <b>recognition</b>, and allowing others to build on your work.
+    <br><br>
+    Openly sharing code and workflows not only fulfills key scientific and ethical principles but also catalyses new discoveries and collaborations across the life sciences. It ensures that research methodologies are transparent, reproducible, and accessible to all, laying the groundwork for a more inclusive and innovative scientific community.
+  </div>
+  <div class="col-5" style="text-align: center;">
+    <img src="/img/illustrations/works-on-my-computer.JPG" alt="Works on my computer" class="img-fluid">
+    <br><small>Robot: MetaManMachine by Nikola Vasiljevic (2021), CC BY-SA 4.0, doi:<a href="https://doi.org/10.5281/zenodo.4471098" target="_blank">10.5281/zenodo.4471098</a></small>
+  </div>
+</div>
+
+### Strengthen scientific integrity and utility
+
+* **Transparency and Reproducibility** – Clear documentation and the ability to replicate analyses build trust in scientific results, and are essential for verifying and understanding research findings.
+* **Reusability and Preservation** – Sharing enables workflows to be applied to new datasets or challenges, maximising the utility and lifespan of research tools and preventing the loss of valuable methodologies.
+
+### Enhance research efficiency and compliance
+
+* **Operational Efficiency** – Modular, well-documented software facilitates smoother research processes, from rerunning experiments to adapting methodologies for new data. 
+* **Legal and Ethical Compliance** – Publishing software and data independently of each other as well as associated research articles simplifies compliance with licensing requirements. Adherence with open code mandates set by journals and funders enhances research credibility and broadens dissemination.
+* **Recognition** – Open code obtains scholarly recognition through citations, emphasising the importance of software as a research output.
```

Comment:
not always just classical citations - can use things like RRID to gain acknowledgement

Reply:
This is 'open code' in the next it's 'open software', maybe stick with 'code' for consistency, as you already defined what 'code' means earlier (so we know it includes software), unless of course something applies specifically to only software